scripts/ubuntu/build_openvslam_docker.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+TOPDIR="$(dirname $(realpath "$0"))"/../../ #openvslam directory
+
+( 
+    cd $TOPDIR
+
+    # Building Docker Image
+    docker build -t openvslam-desktop -f Dockerfile.desktop . --build-arg NUM_THREADS=`expr $(nproc) - 1`
+
+    # Starting Docker Container
+    # before launching the container, allow display access from local users
+    xhost +local:
+    # launch the container
+    GPU_OPTION=
+    if command -v nvidia-container-toolkit &> /dev/null ; then
```

Comment:
Can you use `command -v nvidia-container-toolkit` to check if nvidia-container-toolkit is already installed?
How about using `dpkg -l nvidia-container-toolkit`?

Reply:
Yes, it can check.
"command" is a build in command and "-v" option prints a description of following command. You can see the help by typing "help command" in your shell.
Because "command" is a builtin command, it works not only debian but also other distribution unlike "dkpg".
And this "build_openvslam_docker.sh" script doesn't use any debian specific command, so it can be used in other distribution, so I think it is preferable.

p.s. "which" command also good in this situation.

Reply:
I think `nvidia-container-toolkit` is a package name, but is there really a command with the same name?

Reply:
Yes, it is also a command name.
I already checked this script works.
And here is my output for nvidia-container-toolkit command.
---
$ nvidia-container-toolkit 
Usage of nvidia-container-toolkit:
  -config string
    	configuration file
  -debug
    	enable debug output

Commands:
  prestart
        run the prestart hook
  poststart
        no-op
  poststop
---